docs/joss_paper/paper.md

```diff
@@ -55,67 +67,67 @@ bibliography: paper.bib
+Weak supervision which utilises segmented corresponding regions in the medical image pairs was first proposed in a multimodal application for registering prostate MR to ultrasound images [@hu2018label]. In addition to the regularisation on the predicted displacement field, the training is driven by minimising the dissimilarity between the fixed labels and the warped moving labels. This formulation is modality-independent and the label dissimilarity measures are adapted from well-studied computer vision and medical imaging tasks, such as image segmentation. When multiple corresponding label pairs are available, a two-stage sampling strategy can be adopted for an unbiased gradient back-propagation during training [@hu2018weakly]. \autoref{fig:weakly} shows a schematic illustration of the weakly-supervised network training. Label dissimilarity measures including Dice, Jaccard, square-mean-difference, cross-entropy and their variants. DeepReg uses as a default multiscale Dice [@hu2018weakly], whilst other dissimilarity measures are implemented for users to tailor training options for various applications.
 
-![Registration network with weak supervision loss.\label{fig:weakly}](https://github.com/DeepRegNet/DeepReg/blob/master/docs/source/_images/registration-ddf-nn-weakly-supervised.svg?raw=true)
+![Registration network with weak supervision loss.\label{fig:weakly}](../source/_images/registration-ddf-nn-weakly-supervised.png)
 
 ## Combining unsupervised loss with weak supervision
 Combining the unsupervised loss and the weak supervision has shown superior registration accuracy, compared with that using unsupervised loss alone [@balakrishnan2019voxelmorph]. As a result, the overall loss is the weighted sum of the image dissimilarity, the label dissimilarity and the deformation regularisation, which constrains smoothness of the predicted transformation and is described in the next section. \autoref{fig:combined} shows a schematic illustration of the weakly-supervised network training with unsupervised loss.
```

Comment:
"Combining the unsupervised loss and the weak supervision" -> "Combining the unsupervised loss and weak supervision", remove the second the

Reply:
"constrains smoothness of the xxx" -> "which encourages the smoothness of xxx", as it should constrain an undesired behaviour but we want smoothness